gumroad/README.txt

```diff
@@ -2,30 +2,32 @@
 Contributors: karloscarweber, pderksen, nickyoung87, gumroad
 Tags: gumroad, gumroad product pages, gumroad overlay, gumroad embed, ecommerce, e-commerce, pdf, javascript, overlay, embed
 Requires at least: 3.9
-Tested up to: 4.9.6
-Stable tag: 2.1
+Tested up to: 5.7.1
+Stable tag: 3.0.0
```

Comment:
I have bumped up the major version since this looks like a major change although I have ensured that it won't break the previously inserted Gumroad Blocks as well as shortcodes.

I have done a similar change in the `package.json` manifest, too. 
Not sure if we should bump it in this PR itself or after releasing it on the WordPress plugin marketplace.